lib/req_ch.ex

```diff
@@ -1,12 +1,167 @@
 defmodule ReqCh do
   @moduledoc """
   A Req plugin for ClickHouse.
+
+  By default, `ReqCh` will use TSV as the default output format.
```

Comment:
nitpick: I'd would call it `ReqCH`, we shorten GitHub to GH so we should shorten ClickHouse to CH.